src/DurableTask.SqlServer.AzureFunctions/DurableTask.SqlServer.AzureFunctions.csproj

```diff
@@ -4,7 +4,7 @@
   <Import Project="../common.props" />
   
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
I need to update the TFM here to successfully build, as the new Webjobs.Extensions.DurableTask v3 only supports .NET 6. However, I'm wondering if we consider updating TFM for mssql here.

Reply:
I'm fine with this. 👍